# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/dump/hotfix-static.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- dump-0.4b55.orig/compat/include/bylabel.h	2026-07-26 07:08:44.125892536 +0200
+++ dump-0.4b55/compat/include/bylabel.h	2026-07-26 07:08:47.384225648 +0200
@@ -22,11 +22,11 @@
 #include <blkid/blkid.h>
 #include <stdio.h>
 
-inline const char * get_device_name(const char * item) {
+static inline const char * get_device_name(const char * item) {
 	return blkid_get_devname(NULL, item, NULL);
 }
 
-inline const char * get_device_label(const char * spec) {
+static inline const char * get_device_label(const char * spec) {
 	return blkid_get_tag_value(NULL, "LABEL", spec);
 }
 
